.kitchen.yml

```diff
@@ -0,0 +1,71 @@
+# vim: ft=yaml
+---
+driver:
+  name: vagrant
+  customize:
+    memory: 512
+  vm_hostname: postfix.ci.local
+
+driver_config:
+  require_chef_omnibus: false
+
+provisioner:
+  name: salt_solo
+  # As of 2016/06 http://bootstrap.saltstack.org do not handle ubuntu 16.04.
+  salt_bootstrap_url: https://raw.githubusercontent.com/saltstack/salt-bootstrap/develop/bootstrap-salt.sh
+  salt_install: bootstrap
+  salt_version: latest
+  formula: postfix
+  log_level: info
+  state_top:
+    base:
+      "*":
+        - postfix
+  pillars:
+    top.sls:
+      base:
+        "*":
+          - postfix
+    postfix.sls:
+      server:
+        ssl:
+          enable: false
+
+
+verifier:
+  #name: serverspec
+  test_base_root: tests
+
+
+platforms:
+  - name: bento/ubuntu-14.04
+#  - name: bento/ubuntu-16.04
+#  - name: bento/debian-jessie
+#  - name: bento/centos-7.1
+
+suites:
+
```

Comment:
will remove jessie, but all other are about to be enable as supported platforms. Thus will keep them so other devs may just uncoment.